config.schema.json

```diff
@@ -166,6 +207,26 @@
     "subtext": {
       "type": "string",
       "description": "The subtext shown on the banner"
+    },
+    "start": {
+      "type": "string",
+      "description": "The start time of perp pool promo",
+      "pattern": "^\\d{4}(-\\d\\d(-\\d\\d(T\\d\\d:\\d\\d(:\\d\\d)?(\\.\\d+)?(([+-]\\d\\d:\\d\\d)|Z)?)?)?)?$"
+    },
+    "end": {
+      "type": "string",
+      "description": "The end time of perp pool promo",
+      "pattern": "^\\d{4}(-\\d\\d(-\\d\\d(T\\d\\d:\\d\\d(:\\d\\d)?(\\.\\d+)?(([+-]\\d\\d:\\d\\d)|Z)?)?)?)?$"
+    },
+    "perpPoolDepositBoost": {
+      "type": "string",
+      "description": "The spin boost multiplier for deposits in perp pool",
+      "pattern": "^\\d+$"
+    },
```

Comment:
Think `perpPoolDepositBoost` would be better as a number rather than a number string, then don't need to use `parseInt` on it.